tools/ci/success-http-headers.template.txt

```diff
@@ -9,11 +9,21 @@ x-frame-options: DENY
 x-content-type-options: nosniff
 x-xss-protection: 1; mode=block
 
+HTTP/2 302 
+server: {nginx_version_string}
+content-type: text/html; charset=utf-8
+location: /accounts/go/
+vary: Accept-Language, Cookie
+content-language: en
+strict-transport-security: max-age=15768000
+x-frame-options: DENY
+x-content-type-options: nosniff
+x-xss-protection: 1; mode=block
+
```

Comment:
This should be squashed into the commit that changes the actual behavior to require it.